.circleci/config.yml

```diff
@@ -52,12 +52,25 @@ jobs:
             - packages/*/build
 
   run-non-playground-tests:
-    <<: *defaults
+    docker:
+      - image: circleci/node:10.15.3
+        environment:
+          POSTGRES_HOST: 127.0.0.1
+          POSTGRES_USER: postgres
+          POSTGRES_DATABASE: postgres
+          POSTGRES_PORT: 5432
+
+      - image: circleci/postgres:9.6.2-alpine
```

Comment:
Why are there two images?

Reply:
The second image runs in the background and provides a Postgres server.